### DISCUSSION TITLE
Dagster 1.7.6  + DBT breaks moving from 1.7.X to 1.8.X

### DISCUSSION THREAD

**Discussion by joshua-janicas** [dagster-io/dagster#22060] [Q&A] [answered]
Hello everyone.

I am currently trying to use the latest Dagster (1.7.6) - and while it is functional for DBT in V1.7, the moment I try to update my DBT to 1.8 I get the following error related to Dagster being unable to load modules:

![image](https://github.com/dagster-io/dagster/assets/84326247/7d0a75f7-3b7f-446c-8284-b99e695b414d)

My current setup is as follows (using Python 11):
![Screenshot 2024-05-23 122733](https://github.com/dagster-io/dagster/assets/84326247/b6518ef3-d54e-47dd-a7eb-439530404b42)

Dagster does materialize my assets and is able to run for 1.7, but the moment i change my pip_urls to start using 1.8, the error appears as in the first screenshot. DBT V1.8 does work locally and in my container so I think it's not a DBT issue per say.

Any thoughts or recommendations as to how I can resolve this? Or do I just need to wait for Dagster to be compatible with DBT 1.8?

**Comment by garethbrickman** [comment, marked as answer]
Hey @joshua-janicas, this is resolved in https://github.com/dagster-io/dagster/pull/21908. Support for dbt-core==1.8.* will come out in dagster==1.7.7 and dagster-dbt==0.23.7 later today!

**Reply by joshua-janicas** [reply]
Thank you! 1.7.7 is 👍🏼
